src/components/StripReports.js

```diff
@@ -25,10 +25,20 @@ const StripReports = observer(({ assembly }) => (
                 <Info>{assembly.translate("progress.no_strip_reports")}</Info>
               </Answer>
 
-            : assembly.participant_account.information.strip_reports.map(({created_at, photo, status}) => (
+            : assembly
+                .participant_account
+                .information.strip_reports
+                .slice()
```

Comment:
What does this `.slice()` do?

Reply:
From  - https://mobx.js.org/refguide/array.html

> Unlike the built-in implementation of the functions sort and reverse, observableArray.sort and reverse will not change the array in-place, but only will return a sorted / reversed copy. From MobX 5 and higher this will show a warning. It is recommended to use array.slice().sort() instead.

This is the error, it essentially doesn't sort the array in place. Not sure how that effects our apps performance.